README.md

```diff
@@ -1,22 +1,174 @@
+
+```bash
+chisel cut --release ubuntu-22.04 --root myrootfs/ libgcc-s1_libs libssl3_libs
 ```
 
-## Example release configuration
+In this example, Chisel would look into the Ubuntu Jammy archives, fetch the
+provided packages and install only the desired slices into the *myrootfs*
+folder, according to the slice definitions available in the
+["ubuntu-22.04" chisel-releases branch](<https://github.com/canonical/chisel-releases/tree/ubuntu-22.04>).
+
+## Reference
+
+### Chisel commands
+
+#### help
+
+Show basic help summary. Can also be used to display command details.
+
+#### version
+
+Displays the Chisel version.
+
+#### cut
+
+Uses the provided selection of package slices, for the given release, to create
+a new filesystem tree in the root location.
+
+### Chisel releases
+
+As mentioned above, Chisel relies on **Package Slices**. These slices need to
+be defined prior to the execution of the `chisel` command.
+
+By default, Chisel will look into its central [chisel-releases](https://github.com/canonical/chisel-releases)
+database, where package slices are defined and indexed by Ubuntu release.
+
+One can, however, also point Chisel to a custom and local Chisel release (i.e.
+`chisel cut --release ubuntu-22.10 ...` will fetch the package slices from
+the upstream central database of slices, whilst
+`chisel cut --release release/ ...` will make Chisel fetch the package slices
+from the local path "./release/").
+
+#### Chisel release configuration
+
+Each Chisel release must have one "chisel.yaml" file.
+
+*chisel.yaml*:
+
+```yaml
+format: <chiselReleaseFormat>
+
+archives:
+    ubuntu:
+        # Ubuntu archive for Chisel to look into
+        version: <ubuntuRelease>
+
+        # categories/components of the Ubuntu archive to look into
+        components: [<componentName>, ...]
+
+        # pockets/suites of the Ubuntu archive to look into
+        suites: [<pocket>, ...]
+```
+
+Example:
 
-**release/chisel.yaml**
 ```yaml
 format: chisel-v1
 
 archives:
     ubuntu:
         version: 22.04
         components: [main, universe]
+        suites: [jammy, jammy-security, jammy-updates]
 ```
 
-**release/slices/mypkg.yaml**
+#### Slice definitions
+
+There can be only **one slice definitions file** for each Ubuntu package, per
+Chisel release. All of the slice definitions files must be placed under a
+"slices" folder, and follow the same structure:
+
+*slices/\<pkgName\>.yaml*:
+
+```yaml
+# (req) Name of the package.
+# The slice definition file should be named accordingly (eg. "openssl.yaml")
+
+package: <package-name>
+
+# (req) List of slices
+slices:
+
+    # (req) Name of the slice
+    <slice-name>:
+
+        # (opt) Optional list of slices that this slice depends on
+        essential:
+          - <pkgA_slice-name>
+          - ...
+
+        # (req) The list of files, from the package, that this slice will install
```

Comment:
Should we also mention that the paths in one slice cannot conflict with any other slice in the release?

Reply:
I'd rather not do that before we have a clear strategy on how to address that